wait-for-it.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
I changed this because somebody reported #65, and I know Windows does not include /usr/bin/env, so I'm trying to see if that is going to fix it.